.github/workflows/build.yaml

```diff
@@ -36,8 +36,6 @@ jobs:
       - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
 
       - uses: sigstore/cosign-installer@c3667d99424e7e6047999fb6246c0da843953c65
```

Comment:
can you also check if this hash is from the latest release?

Reply:
yes it is

Reply:
oh, it got merged a new PR 23min ago.

Reply:
thanks